tests/views/test_collection_exercise.py

```diff
@@ -536,7 +540,7 @@ def test_view_collection_instrument(self, mock_request, mock_details):
     @patch('response_operations_ui.views.collection_exercise.build_collection_exercise_details')
     def test_choose_collection_instrument_when_first(self, mock_request, mock_details):
         with open(
-            "tests/test_data/collection_exercise/formatted_collection_exercise_details_no_ci.json"
+                "tests/test_data/collection_exercise/formatted_collection_exercise_details_no_ci.json"
```

Comment:
What's with the odd spacing?  It'd be better to have it all on 1 line.  If can, change this for all of them that you've changed in this file.

Reply:
This is the ESlint, don't think its a harm looks good.